lib/tasks/foreman_templates_tasks.rake

```diff
@@ -2,6 +2,7 @@
 namespace :templates do
   desc 'Import templates according to settings'
   task :import => :environment do
+    puts "You are using a deprecated behavior, it will be removed in a future version. Please use appropriate API endpoint for this functionality"
```

Comment:
Cant we set this as part of logging, instead of just plain print statement:
e.g : 
```
2018-02-26T13:59:42  [app] [W] You are using a deprecated behavior, it will be removed in a future version. Please use appropriate API endpoint for this functionality.
```
Currently, It's just a plain statement.